ClosedXML/Excel/ConditionalFormats/XLConditionalFormat.cs

```diff
@@ -39,6 +39,9 @@ public XLConditionalFormat(XLConditionalFormat conditionalFormat)
             ReverseIconOrder = conditionalFormat.ReverseIconOrder;
             ShowIconOnly = conditionalFormat.ShowIconOnly;
             ShowBarOnly = conditionalFormat.ShowBarOnly;
+            StopIfTrueInternal = OpenXmlHelper.GetBooleanValueAsBool(conditionalFormat.StopIfTrueInternal, true);
```

Comment:
This doesn't make sense. OpenXML types, e.g. `BooleanValue` should not be used here.

Reply:
good catch, sorry about that. Bad copy paste.